### ISSUE TITLE
Would be cool if it could open a url in another tab when used from wasm

### ISSUE THREAD

**Issue by gilescope** [Byron/open-rs#49]
Would be very cool if it could open a url in another tab when used from a wasm app.

**Comment by Byron** [comment]
Should I be worried if I wasn't even aware that `open::that(…)` works in the browser?
Unless I am missing something, it looks like the prerequisite for this is WASM support, maybe that's something you would be interested in?

**Comment by hoodie** [comment]
should be relatively straight forward to embed some javascript calls

**Comment by gilescope** [comment]
Yes I guess wasm support would be a prerequisite. Would be pretty darn cool
if it just worked whether you made a desktop app or a browser app.
